android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/home/HomeViewModel.kt

```diff
@@ -42,9 +45,9 @@ class HomeViewModel(
         ).uppercase()
     }
 
-    fun loadProfileAndSurveys() {
+    fun loadProfileAndSurveys(isRefresh: Boolean = false) {
```

Comment:
we lack of a test in HomeViewModelTest to verify the expected result for isRefresh = true 🤔

Reply:
Nice catch! Fixed in https://github.com/suho/kmm-ic/pull/128/commits/756ce645011023fee42efd9bc3ffd8447309170a